app/src/main/java/net/gsantner/markor/util/StringUtils.java

```diff
@@ -0,0 +1,54 @@
+package net.gsantner.markor.util;
+
+public final class StringUtils {
```

Comment:
We seem to be writing these routines over and over again. Moved them all here.

This module ought to be tested

Reply:
Can you move this to the opoc/util package?